doc/api/fs.md

```diff
@@ -1519,6 +1519,9 @@ changes:
   - version: v2.3.0
     pr-url: https://github.com/nodejs/node/pull/1845
     description: The passed `options` object can be a string now.
+  - version: REPLACEME
+    pr-url: https://github.com/nodejs/node/pull/REPLACEME
+    description: `fs` open allows overriding the used fs implementattion.
```

Comment:
If I recall correctly, YAML sections with the first `` ` `` char usually throw in doc building. If so, this usually needs a workaround like this:
```suggestion
    description: The `fs.open()` allows overriding the used `fs` implementation.
```